### PR TITLE
steps-chatwork-message 0.0.3

### PR DESCRIPTION
**Pull request by monolithic-adam**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/steps-chatwork-message/0.0.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  go:
+    package_name: github.com/monolithic-adam/bitrise-step-chatwork-message
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      **An api\_token input is required.**
```
Comment (richard-bogdan-bitrise) [inline steps/steps-chatwork-message/0.0.3/step.yml]: "\\" is not needed.

File: steps/steps-chatwork-message/0.0.3/step.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+      To get your **API token** visit: https://www.chatwork.com/service/packages/chatwork/subpackages/api/token.php
+    is_required: true
+    is_sensitive: true
+    title: Chatwork API Token
+- opts:
+    description: |
+      **An room\_id input is required.**
```
Comment (richard-bogdan-bitrise) [inline steps/steps-chatwork-message/0.0.3/step.yml]: "\\" is not needed.

File: steps/steps-chatwork-message/0.0.3/step.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+      To get your **Room ID** look at the #!rid in your URL
```
Comment (richard-bogdan-bitrise) [inline steps/steps-chatwork-message/0.0.3/step.yml]: Please add more details about this, make it more clear.

File: steps/steps-chatwork-message/0.0.3/step.yml (hunk 1 above)
Comment (monolithic-adam) [reply steps/steps-chatwork-message/0.0.3/step.yml]: fixed!

File: steps/steps-chatwork-message/0.0.3/step.yml (hunk 2 above)
Comment (monolithic-adam) [reply steps/steps-chatwork-message/0.0.3/step.yml]: fixed!

File: steps/steps-chatwork-message/0.0.3/step.yml (hunk 3 above)
Comment (monolithic-adam) [reply steps/steps-chatwork-message/0.0.3/step.yml]: added a bit more detail and a URL for help

**Pull request closed without merging**